manifests/reposync-crd.yaml

```diff
@@ -279,6 +279,20 @@ spec:
                     - gcpserviceaccount
                     - none
                     type: string
+                  caCertSecretRef:
+                    description: caCertSecretRef specifies the name of the secret
+                      where the CA certificate is stored. The creation of the secret
```

Comment:
`The creation of the secret should be done out of band by the user` seems weird to note here. Is there precedent? Maybe just cut it.

Reply:
Yes, this is written similarly to the existing git field